components/table/index.zh-CN.md

```diff
@@ -92,6 +92,8 @@ const columns = [
 | onChange | 分页、排序、筛选变化时触发 | function(pagination, filters, sorter, extra: { currentDataSource: \[], action: `paginate` \| `sort` \| `filter` }) | - |  |
 | onHeaderRow | 设置头部行属性 | function(columns, index) | - |  |
 | onRow | 设置行属性 | function(record, index) | - |  |
+| prefix | 自定义 Table 内部前缀元素 | ReactNode | - |  |
+| suffix | 自定义 Table 内部后缀元素 | ReactNode | - |  |
```

Comment:
I don't think `prefix` and `suffix` are necessary, it is same:

```
{prefix}
<Table />
{suffix}
```

Reply:
That needs a wrapper element, which I'd like to avoid.

Reply:
It is used for

```tsx
<Table prefix={stikcy && <div ref={ref} className="sticky-sentinel"></div>} />
```

Reply:
@afc163 Any further discussion? The added props are just simple, but if you don't want it strongly, I can remove it in this PR.

Reply:
I prefer not adding `prefix` and `suffix`.

Reply:
@afc163 Done!